ethcore/src/test_helpers/evm_test_client.rs

```diff
@@ -110,8 +110,10 @@ impl<'a> EvmTestClient<'a> {
 			ForkSpec::Constantinople => Some(spec::new_constantinople_test()),
 			ForkSpec::ConstantinopleFix => Some(spec::new_constantinople_fix_test()),
 			ForkSpec::Istanbul => Some(spec::new_istanbul_test()),
-			ForkSpec::EIP158ToByzantiumAt5 => Some(spec::new_transition_test()),
+			ForkSpec::EIP158ToByzantiumAt5 => Some(spec::new_eip158_to_byzantiumat5_test()),
 			ForkSpec::FrontierToHomesteadAt5 | ForkSpec::HomesteadToDaoAt5 | ForkSpec::HomesteadToEIP150At5 => None,
+			ForkSpec::ByzantiumToConstantinopleFixAt5 => Some(spec::new_byzantium_to_constantinoplefixat5_test()), 
+			ForkSpec::ConstantinopleFixToIstanbulAt5 => None, // [@adria0] fix
```

Comment:
```suggestion
			ForkSpec::ConstantinopleFixToIstanbulAt5 => None,
```

Reply:
This is not working for the moment... there is a test that the state hash does not match, but the test has something "rare" there, like forcing the value instead of computing it.

https://github.com/ethereum/tests/blob/5841af6da472fb3f19810354cf9a30afd8e72b5f/src/BlockchainTestsFiller/TransitionTests/bcByzantiumToConstantinopleFix/ConstantinopleFixTransitionFiller.json#L115

```
{
   "blocknumber" : "5",
   "blockHeader" : {
       "stateRoot" : "0xb5d97b0160c668ec9711bb223b8043f77302a762b9c3e00780714ef78ae5bbef"
     },
     ...
```

asked for it in the ethereum/tests